backend/ohq/views.py

```diff
@@ -271,10 +271,14 @@ def get_queryset(self):
             .values("count")
         )
 
-        return Queue.objects.filter(course=self.kwargs["course_pk"], archived=False).annotate(
-            questions_active=Subquery(questions_active[:1], output_field=IntegerField()),
-            questions_asked=Subquery(questions_asked[:1]),
-            staff_active=Subquery(staff_active[:1]),
+        return (
+            Queue.objects.filter(course=self.kwargs["course_pk"], archived=False)
+            .annotate(
+                questions_active=Subquery(questions_active[:1], output_field=IntegerField()),
+                questions_asked=Subquery(questions_asked[:1]),
+                staff_active=Subquery(staff_active[:1]),
+            )
+            .order_by()
```

Comment:
This will implicitly order by primary key, i'm pretty sure. We probably would like to explicitly order by `time_asked` or whatever that field is called. But it should work!